stevePObjects/src/carPackage/carClass.java

```diff
@@ -0,0 +1,41 @@
+package carPackage;
+
+/**
+ *Car class
+ * @author Steve
+ */
+public class carClass {
+    //fields
+    public int yearModel, carSpeed;//year of car & speed of car
+    public String carMake;//make of car
+    
+    //constructor
+    public carClass(int yearModel, String carMake){
+        this.yearModel = yearModel;
+        this.carSpeed = 0;
+        this.carMake = carMake;
```

Comment:
Prefixing items here with car could be considered redundant since everything within the class is part of car